src/hammer-vlsi/defaults.yml

```diff
@@ -273,6 +273,8 @@ vlsi.inputs:
   #                          For example, if this hardmacro is an instance of "my_amplifier", this
   #                          field should be set to "my_amplifier".
   # - Required for hierarchical and optional for hardmacro, disallowed otherwise
+  # create (Optional[bool]) - Create an instance that does not exist
```

Comment:
Add "In addition to placing the instance".

Reply:
fixed